### DISCUSSION TITLE
Possible to build only the terminal?

### DISCUSSION THREAD

**Discussion by 0x5c** [wezterm/wezterm#1416] [Q&A] [answered]
I'm on the lookout for a new terminal emulator, and wezterm looks like a good fit for my needs, except that it comes with a full ssh client, a multiplexing server, and other features I'd prefer to keep outside my terminal (using existing solutions).

Is it possible to build only the terminal? I've looked for cargo features to disable but none seem to be related to ssh/etc. Is there another way? or are cargo features on the roadmap?

**Comment by wez** [comment, marked as answer]
No, and I have no plans for that: it would introduce more complexity into the code base and make it harder to maintain.

**Reply by 0x5c** [reply]
Ah that's a bit unfortunate, but thanks for the fast answer
